packages/inter-protocol/src/psm/psmCharter.js

```diff
@@ -0,0 +1,74 @@
+// @ts-check
+
+import '@agoric/governance/src/exported.js';
+import '@agoric/zoe/exported.js';
+import '@agoric/zoe/src/contracts/exported.js';
+
+import { E, Far } from '@endo/far';
+
+/**
+ * @file
+ *
+ * This contract makes it possible for those who govern the PSM to call for
+ * votes on changes. A more complete implementation would validate parameters,
+ * constrain deadlines and possibly split the ability to call for votes into
+ * separate capabilities for finer grain encapsulation.
+ */
+
+/**
+ * @param {ZCF<{binaryVoteCounterInstallation:Installation}>} zcf
+ * @param {{psm:GovernedContractFacetAccess<{},{}>}} privateArgs
+ */
+export const start = async (zcf, privateArgs) => {
+  const { binaryVoteCounterInstallation: counter } = zcf.getTerms();
+  const { psm } = privateArgs;
```

Comment:
This is the powerful governed facet that lets you call for votes. It can't be in terms.

AFAICT, no one has a handle to the charter, so they wouldn't be able to look at its terms or call validation methods on it. As it currently stands, only the committee members (and bootstrap) will end up with references to the charter. The committee members can verify the instance and installation they've been given, and that bootstrap only provides access to the ability to solicit votes by wrapping it in a charter. Everyone else can verify how bootstrap hands it out, and whatever we do to make the committee members identities legible.

Reply:
Ah. I was thinking it was the instance.